ui/v2.5/index.html

```diff
@@ -13,7 +13,7 @@
       manifest.json provides metadata used when your web app is installed on a
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
-    <link rel="manifest" href="/%BASE_URL%/manifest.json" />
+    <link rel="manifest" href="manifest.json" />
```

Comment:
I believe this will break accessing the manifest when a subpath proxy is configured. However, if it does, we might just want to get rid of the manifest / meta tags for A2HS. It's not set up right anyway